cmd/swarm/upload-speed/main.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
@holisticode why do you need this separate `cmd`, when you have already added the function under `app.Commands` for `swarm-smoke`?